workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/GameObjectRepresentation/SpatialOSBehaviourLibraryTests.cs

```diff
@@ -12,16 +12,11 @@ namespace Improbable.Gdk.Core.EditmodeTests
     [TestFixture]
     public class SpatialOSBehaviourLibraryTests
     {
-        private const uint ComponentId1 = 1337;
+        // The dummy code needs to be removed once we have codegenerated writers.
+        private const uint PositionComponentId = 54;
```

Comment:
Could we get this through reflection? Admittedly there should be no reason for it to change, but if it does that could be a bit of a pain. Could still keep readable by having a static method and writing `PositionComponentId = GetIdFor(typeof(Position))` or similar

Reply:
We will probably want to implement something similar to `Dynamic` in the C# SDK

Reply:
What is that?

Reply:
Actually just realised components don't even know their ID at the moment right? In which case there's no easy way to do what I suggested that should be in scope of this PR, so ignore it